_blog-src/_posts/2016-02-22-plugin-mgmt.markdown

```diff
@@ -0,0 +1,260 @@
+
+So, by including a “-SNAPSHOT” suffix in your custom policy plugin’s version, you can iterate through changes to the policy plugin  without having to manually uninstall and then install each new version of the plugin.
+
+Let’s take a look at this in action.
+
+Since we want to take advantage of the automatic reloading, we must restore the “-SNAPSHOT” suffix to the test-policy custom policy plugin. To make these changes, edit these files, and change the version from “1.2.3” to “1.2.2-SNAPSHOT”:
+
+```shell
+./pom.xml
+test-policy/pom.xml
+```
+
+Then, to build the plugins, and install them into your local maven repo, execute this command:
+
+```shell
+mvn clean install
+```
+
+And then add the custom policy plugin in the Management UI:
+
+![Image: Add SNAPSHOT Plugin ](/blog/images/2016-02-22/plugin_mgmt_10.png)
+
+And, here’s the installed plugin:
+
+![Image: View SNAPSHOT Plugin ](/blog/images/2016-02-22/plugin_mgmt_11.png)
+
+Before we can configure the custom policy plugin, we need an API. For this example, we’ll use our old friend, “apiman-echo.” You can download this API from this git repository: [https://github.com/apiman/apiman-quickstarts](https://github.com/apiman/apiman-quickstarts)
+
+The steps to build, deploy, and configure this API are available in the first article in this series: [The Impatient New User Guide to apiman](http://www.apiman.io/blog/introduction/overview/2015/01/09/impatient-new-user-redux.html)
+
+We’ll use the postman web client to access the API. The first time that we access the API, we’ll see this response (note the text highlighted in green):
+
+```json
+{
+ "method" : "GET",
+ "resource" : "/apiman-echo",
+ "uri" : "/apiman-echo",
+ "headers" : {
+   "Accept" : "*/*",
+   "Cache-Control" : "no-cache",
+   "User-Agent" : "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/46.0.2490.86 Safari/537.36",
+   "Connection" : "keep-alive",
+   "Test-Policy" : "true",
+   "Postman-Token" : "8808bb68-1c1c-ef97-449d-ab60f620b0e5",
+   "Host" : "localhost:8080",
+   "Accept-Language" : "en-US,en;q=0.8",
+   "Accept-Encoding" : "gzip, deflate, sdch"
+ },
+ "bodyLength" : null,
+ "bodySha1" : null
+}
+```
+
+OK, now, let’s change the policy plugin. In this file: src/main/java/io/apiman/plugins/test_policy/TestPolicy.java
+
+Change this line from this: request.getHeaders().put("Test-Policy", "true");
+
+To this: request.getHeaders().put("Test-Policy", "quite true");
+
+And then, rebuild the plugin with this command:
+
+```shell
+mvn clean install -DskipTests
+```
+
+(The “skipTests” directive is a bit of laziness. There is a test included in the plugin that will fail because of the change that we just made. You can either run this command as it, or you can edit the test in the plugin to also look for a string of “quite true.”)
```

Comment:
Good catch.  :)  I'll merge and then fix it so that len doesn't have to update the PR.